client/src/Components/WhatTheySay/index.jsx

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable import/no-unresolved */
+import React from 'react';
+import { Grid, Row, Col } from 'antd';
+import { Pagination } from 'swiper';
+import { Swiper, SwiperSlide } from 'swiper/react';
+import 'swiper/css/bundle';
+import 'swiper/css';
+import 'swiper/css/pagination';
+import QuoteCard from '../QuoteCard';
+import './style.css';
+import CustomTitle from '../CustomTitle';
+
+const { useBreakpoint } = Grid;
+function WhatTheySay() {
+  const screens = useBreakpoint();
+  let count = 3;
+  if (screens.xl) count = 3;
+  else if (screens.sm) count = 2;
+  else if (screens.xs) count = 1;
+
+  const quotes = [
+    {
+      name: 'محمد',
+      desc: 'دائما يقومون بعمل جيد وبالوقت المحدد, انا حقا سعيد بهذه التجربة',
+      img: 'https://images.unsplash.com/photo-1599566150163-29194dcaad36?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=387&q=80',
+    },
+    {
+      name: 'اسراء',
+      desc: 'دائما يقومون بعمل جيد وبالوقت المحدد, انا حقا سعيد بهذه التجربة',
+      img: 'https://images.unsplash.com/photo-1494790108377-be9c29b29330?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=387&q=80',
+    },
+    {
+      name: 'جهاد',
+      desc: 'دائما يقومون بعمل جيد وبالوقت المحدد, انا حقا سعيد بهذه التجربة',
+      img: 'https://images.unsplash.com/photo-1487412720507-e7ab37603c6f?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=871&q=80',
+    },
+    {
+      name: 'مصطفى',
+      desc: 'دائما يقومون بعمل جيد وبالوقت المحدد, انا حقا سعيد بهذه التجربة',
+      img: 'https://images.unsplash.com/photo-1552374196-c4e7ffc6e126?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=387&q=80',
+    },
+  ];
```

Comment:
Move these to an external `Quotes.json` file so if the administrator -that they might not be developers- wants to edit the quotes or adding a new quotes they wouldn't touch the frontend code and ruin it.